src/schema/value.rs

```diff
@@ -32,6 +33,8 @@ pub enum Value {
     Bytes(Vec<u8>),
     /// Json object value.
     JsonObject(serde_json::Map<String, serde_json::Value>),
+    /// Ip
+    Ip(IpAddr),
```

Comment:
We should enforce (and document) normalization whenever we build an IpAddr address.

Here that means 
```suggestion
    Ip(Ipv6Addr),
```

Reply:
and mapping Ip v4 to Ip V6 using mapped IpV4 address.

(I don't think there are other normalziation to do, but you can dig further)

Reply:
We lose some info here (was this IPv4 address supplied in ipv4 format or ipv6 mapped) but this is a good thing. We will have to day that for search anyway.

Reply:
Currently we do the normalization internally when converting to u128. 
I'm not sure about the advantage of putting it into the API?